tests/ssl-opt.sh

```diff
@@ -1827,6 +1840,20 @@ else
     }
 fi
 
+# Filter tests according to TEST_SUITE_NAME
+is_excluded_test_suite () {
+    if [ -n "$RUN_TEST_SUITE" ]
+    then
+        case ",$RUN_TEST_SUITE," in
+            *",$1,"*) false;;
```

Comment:
I do not think so.

When known test suites are ` a b c d` and `$1` is `ab`, we should not run any test suite. but your suggestion will run suite `a` and `b`

Reply:
> But reconsidering that adding `,,` is no harm

The commas are must. Considerring that `RUN_TEST_SUITE` is `ab`, and we have suites `a`, `b`, `c`, `d`. If there was no commas, when testing suite `a`, `$1` should be `a`, and `ab` would match the pattern `*a*`, which would cause suite `a` to be excuted. This is not the expected behaviour. You could see similar case [here](https://github.com/Mbed-TLS/mbedtls/blob/8b6b41f6cd86c70ea92d3ce45b3479b54364f2c8/tests/scripts/all.sh#L519-L524).